templates/content/node--oe-news--teaser.html.twig

```diff
@@ -10,7 +10,8 @@
   'url': url,
   'title': content.extra_field_oe_theme_helper_short_title_with_fallback|field_value,
   'meta': [
-    content.oe_publication_date|field_value
+    content.news_types|upper,
+    content.date|upper
```

Comment:
Remove `|upper`, we will fix this in CSS later on. Also here use the actual fields.